Gibe.DittoProcessors/packages.config

```diff
@@ -39,6 +39,7 @@
   <package id="Owin" version="1.0" targetFramework="net451" />
   <package id="semver" version="1.1.2" targetFramework="net451" />
   <package id="SharpZipLib" version="0.86.0" targetFramework="net451" />
+  <package id="System.ValueTuple" version="4.3.0" targetFramework="net452" />
```

Comment:
MediaService line 102, used to couple a media item together with whether it is a folder or not